### PR TITLE
Clean-up grabberDual

### PR DESCRIPTION
**Pull request by martinaxgloria**
As per title, this PR contains the cleaning up of the leftovers of `grabberDual` service.

### CODE REVIEW

File: experimentalSetups/head_face-v3/camera/dragonfly2_config_left.ini [hunk 1]
```diff
@@ -1,4 +1,5 @@
 device dragonfly2
+capabilities RAW
```
Comment (Nicogene) [inline experimentalSetups/head_face-v3/camera/dragonfly2_config_left.ini:2]: For `dragonfly2` the parameter `capabilities` as to be set to `COLOR` or not set (`COLOR` is default)

**Review by Nicogene — CHANGES_REQUESTED** [review]

**Comment by pattacini** [comment]
We need to address also `icub-main` apps to replace references to the dual configurations.

**Comment by martinaxgloria** [comment]
Ok @pattacini tomorrow I will check for the other references in icub-main.

**Comment by martinaxgloria** [comment]
I opened the PR for the clean-up of `dualGrabber` references in icub-main.

- https://github.com/robotology/icub-main/pull/861

File: iCubGenova11/camera/this doesn't needed/ServerGrabberDualDragon.ini
Comment (pattacini) [inline iCubGenova11/camera/this doesn't needed/ServerGrabberDualDragon.ini:1]: This file needs to be removed outright.

File: iCubGenova11/camera/this doesn't needed/ServerGrabberDualDragonBayer640_480.ini
Comment (pattacini) [inline iCubGenova11/camera/this doesn't needed/ServerGrabberDualDragonBayer640_480.ini:1]: This file needs to be removed outright.

**Review by pattacini — CHANGES_REQUESTED** [review]
Other files to be removed.

File: iCubGenova11/camera/this doesn't needed/ServerGrabberDualDragon.ini
Comment (martinaxgloria) [reply iCubGenova11/camera/this doesn't needed/ServerGrabberDualDragon.ini:1]: Ok, I will remove the entire folder then.

**Comment by Nicogene** [comment]
> If I interpreted @Nicogene's comment correctly, all `capabilities` need to be replaced from `RAW` to `COLOR`.

We aligned T2T, for `dragonfly2` is ok also to not specify `capabilities`, since `COLOR` is the default.

File: experimentalSetups/head_face-v3/camera/dragonfly2_config_right.ini [hunk 2]
```diff
@@ -1,4 +1,6 @@
 device dragonfly2
+capabilities COLOR
+capabilities RAW
```
Comment (Nicogene) [inline experimentalSetups/head_face-v3/camera/dragonfly2_config_right.ini:3]: Probably a find/replace error, `capabilities RAW` has to be removed here and for the other `dragonfly2`

**Review by Nicogene — CHANGES_REQUESTED** [review]

**Review by Nicogene — APPROVED** [review]
Obviously it is impossible to check all the files, but they seems ok overall now 👍🏻 

We need a test on a robot before merging

**Review by pattacini — APPROVED** [review]

**Review by Nicogene — CHANGES_REQUESTED** [review]
@pattacini good catch!

> What's the strategy with R1?

We should change `grabberDual` with `frameGrabber_nws_yarp` 

cc @martinaxgloria

**Comment by martinaxgloria** [comment]
Today I tested the changes on `iCubGenova11` and I was able to open both cameras. 

photo_5870860122425769501_w

**Review by Nicogene — APPROVED** [review]

**Pull request merged**